mozilla-release/toolkit/components/extensions/webrequest/WebRequest.jsm

```diff
@@ -723,6 +723,25 @@ HttpObserverManager = {
         this.handleForgetModeNotification(otherWin);
       }
 
+      try {
+        // Covers the case of balnk window in case explicit link is opened in new tab from context menu
+        const { BrowserWindowTracker } = ChromeUtils.import("resource:///modules/BrowserWindowTracker.jsm");
+        BrowserWindowTracker.orderedWindows.forEach(w => {
+          const { tabs } = w.gBrowser;
+          if (
+              (
+                tabs[0]._fullLabel === originURL || // for cases like google
+                !tabs[0]._fullLabel // for cases like duckduckgo
+              ) &&
+              tabs.length === 1
```

Comment:
@tsl143 also could you please tell what happens if a browser has more than 1 tab?
For example, porno-movie.domain will still be opened in the original tab and also in a forget window?

Reply:
In this case there must not be second tab, as we will land in this situation only when user opens it via `open in new window`

Reply:
for version tag, lets discuss this offline as this is beyond scope of this PR

Reply:
> for version tag, lets discuss this offline as this is beyond scope of this PR

Sure.